src/components/SwapForm/AddMEVProtectionModal.tsx

```diff
@@ -113,10 +112,9 @@ export default function AddMEVProtectionModal({ isOpen, onClose }: { isOpen: boo
         onClose?.()
         mixpanelHandler(MIXPANEL_TYPE.MEV_ADD_RESULT, { type: addingOption.name, result: 'success' })
       },
-      (connector: Connector, error: Error) => {
-        let reason = error?.message || 'Unknown reason'
-        if (didUserReject(connector, error)) reason = 'User rejected'
-        mixpanelHandler(MIXPANEL_TYPE.MEV_ADD_RESULT, { type: addingOption.name, result: 'fail', reason })
+      (error: Error) => {
+        const message = friendlyError(error)
+        mixpanelHandler(MIXPANEL_TYPE.MEV_ADD_RESULT, { type: addingOption.name, result: 'fail', reason: message })
```

Comment:
should keep raw message on mixpanel. only need friendly message for end user

Reply:
This is request from data analyst